integration-tests/reactive-mysql-client/README.md

```diff
@@ -4,7 +4,7 @@
 
 By default, the tests of this module are disabled.
 
-To run the tests in a standard JVM with MySQL started as a Docker container, you can run the following command:
+To run the tests in a standard JVM with MariaDB started as a Docker container, you can run the following command:
```

Comment:
This module is called `reactive-mysql-client`: why is it using MariaDB for testing?

Reply:
Because I suppose it supports both? I haven't changed it, just made the README state what the code does. @tsegismont wrote this IIRC.